From 5ed27dbdbfe866810a52ff8225bcf61590861823 Mon Sep 17 00:00:00 2001
From: Frederic Crozat <fcrozat@suse.com>
Date: Thu, 18 Aug 2011 13:50:43 +0000
Subject: PATCH: add missing header include

Hi,

MS_REC is not always defined in sys/mount.h. linux/fs.h should be
included, since it is always defined there.
--
Frederic Crozat <fcrozat@suse.com>
SUSE

>From 9f8a50decf45aaa4436b9fb3a0ab360f238b7d96 Mon Sep 17 00:00:00 2001
From: Frederic Crozat <fcrozat@suse.com>
Date: Thu, 18 Aug 2011 15:42:29 +0200
Subject: [PATCH] nspawn: add missing include header for MS_REC.
---
diff --git a/src/nspawn.c b/src/nspawn.c
index 2c1144a..19d95b2 100644
--- a/src/nspawn.c
+++ b/src/nspawn.c
@@ -37,6 +37,7 @@
 #include <termios.h>
 #include <sys/signalfd.h>
 #include <grp.h>
+#include <linux/fs.h>
 
 #include "log.h"
 #include "util.h"
--
cgit v0.9.0.2-2-gbebe
